src/commands/app/db/provision.js

```diff
@@ -36,36 +37,62 @@ export class Provision extends DBBaseCommand {
 
           return {
             status: 'already_provisioned',
             namespace: this.rtNamespace,
-            region: provisionStatusResponse.region || 'amer',
+            region: provisionStatusResponse.region || DEFAULT_REGION,
             details: provisionStatusResponse
           }
-        } else if (currentStatus === 'REQUESTED' || currentStatus === 'PROCESSING') {
-          this.log(chalk.yellow('Database provisioning is already in progress'))
+        } else if (currentStatus === DB_STATUS.REQUESTED) {
+          this.log(chalk.yellow('Database provisioning request has been submitted and is pending'))
           this.log(chalk.dim(`   Namespace: ${this.rtNamespace}`))
-          this.log(chalk.dim(`   Region: ${provisionStatusResponse.region || 'amer'}`))
+          this.log(chalk.dim(`   Region: ${provisionStatusResponse.region || DEFAULT_REGION}`))
           this.log(chalk.dim(`   Status: ${currentStatus}`))
           this.log(chalk.dim('\nUse "aio app db status --watch" to monitor progress'))
 
           return {
             status: 'in_progress',
             namespace: this.rtNamespace,
-            region: provisionStatusResponse.region || 'amer',
+            region: provisionStatusResponse.region || DEFAULT_REGION,
             details: provisionStatusResponse
           }
-        } else if (currentStatus === 'FAILED') {
+        } else if (currentStatus === DB_STATUS.PROCESSING) {
+          this.log(chalk.yellow('Database is currently being provisioned'))
+          this.log(chalk.dim(`   Namespace: ${this.rtNamespace}`))
+          this.log(chalk.dim(`   Region: ${provisionStatusResponse.region || DEFAULT_REGION}`))
+          this.log(chalk.dim(`   Status: ${currentStatus}`))
+          this.log(chalk.dim('\nUse "aio app db status --watch" to monitor progress'))
+          this.log(chalk.red('If provisioning takes unusually long, please contact the App Builder team'))
+
+          return {
+            status: 'in_progress',
+            namespace: this.rtNamespace,
+            region: provisionStatusResponse.region || DEFAULT_REGION,
+            details: provisionStatusResponse
+          }
+        } else if (currentStatus === DB_STATUS.FAILED) {
           this.log(chalk.red('Previous database provisioning failed'))
           this.log(chalk.dim(`   Namespace: ${this.rtNamespace}`))
           this.log(chalk.dim(`   Status: ${currentStatus}`))
           this.log(chalk.yellow('\nAttempting to provision again...'))
+          this.log(chalk.red('If the problem persists, please contact the App Builder team'))
+        } else if (currentStatus === DB_STATUS.REJECTED) {
+          this.log(chalk.red('Previous database provisioning request was rejected'))
+          this.log(chalk.dim(`   Namespace: ${this.rtNamespace}`))
+          this.log(chalk.dim(`   Status: ${currentStatus}`))
+          this.log(chalk.yellow('\nYou can attempt to provision again, but the request may be rejected again'))
```

Comment:
```suggestion
          this.log(chalk.yellow('\nAttempting to provision again...'))
```

I don't like the "You can attempt" part of this because it sounds like they need to do something additional to submit another request, but the code below is doing it already.

I'm also unsure if we should allow submitting a request if the previous one was rejected, as we may want to force them to resolve the reason why it was rejected first, but we can address that when we decide on how the final provisioning process works.